File: 250413/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    ios::sync_with_stdio(false);
    cin.tie(0);
    cout.tie(0);

    string str[5] = {"apple", "banana", "grape", "blueberry", "orange"};
    string s;
    int count = 0;
    cin >> s;
    for(int i = 0; i < 5; i++) {
        if(str[i][2] == s[0] || str[i][3] == s[0]) {
            cout << str[i] << "\n";
            count++;
        }
    }
    cout << count;
    return 0;
}

File: 250413/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        String[] str = new String[] { "apple", "banana", "grape", "blueberry", "orange" };
        String s = sc.next();
        int count = 0;
        for(int i = 0; i < 5; i++) {
            if(s.charAt(0)== str[i].charAt(2) || s.charAt(0) == str[i].charAt(3)) {
                System.out.println(str[i]);
                count++;
            }
        }
        System.out.println(count);
        sc.close();
    }
}